docs/security_groups.md

```diff
@@ -0,0 +1,55 @@
+# Security Groups
+
+## Use existing AWS Security Groups
+**Note: Use this at your own risk, when existing SGs are used Kops will NOT ensure they are properly configured.**
```

Comment:
Yep, this already happens. The issue is that this displays a compressed object on the terminal and unless you are looking for it, it can be easy to miss. Hence the note here, however I'm fine with removing the note as well.